opal/browser/audio/param_schedule.rb

```diff
@@ -0,0 +1,24 @@
+module Browser; module Audio
+
+  class ParamSchedule
+    include Native
+
+    alias_native :value,               :setValueAtTime
```

Comment:
@meh not sure about using `=` in this one :confused: , `setValueAtTime` takes two parameters...